stripper/ze_thelostworld_redux_p2.cfg

```diff
@@ -1,3 +1,47 @@
+;Make it so players spawning late (usually due to a suicide) don't get a boost out of the plane they can use to strafe in water with >200 speed to early trigger bridge and boat.
+;Speed change is because plane moves closer to island far faster than players can swim, leading to late spawns being 20-30 secs ahead of other players.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_push"
+		"targetname" "SpawnPlane_Push"
+	}
+	insert:
+	{
+		"OnUser1" "!activatorRunScriptCodeself.SetVelocity(Vector(0,0,-200));0-1"
+		"OnUser1" "speedModifySpeed0.50-1"
+		"OnUser1" "speedModifySpeed120-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "SpawnPlane_PushAddOutputOnEndTouch SpawnPlane_Push,FireUser1,,0.05,-112.5-1"
```

Comment:
Have you tested this? I've never seen AddOutput syntax used with commas instead of colons before.

Reply:
Yup, it works fine. Used in a few other strippers I made too, like otakuroom: https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/42e961b1d1320761a698d5194b10ae9cd3d789b2/stripper/ze_otakuroom_v5_6f3.cfg#L130